dali/pipeline/data/tensor_vector.h

```diff
@@ -59,6 +60,10 @@ class TensorVector {
     return *(tensors_[pos]);
   }
 
+  const std::vector<std::shared_ptr<Tensor<Backend>>> &tensors() const {
```

Comment:
I am not sure if we want to expose the underlying vector, the intention was to probably merge the `TensorList` and `TensorVector` some time in the future. I see you use it mostly for the copy to `TensorList`, you can probably add an overload for `TensorVector` in the `Copy`

@mzient, what do you think?

Reply:
I agree - better to have a TensorVector overload in TensorList.

Reply:
done